src/common/ColourPalette.cpp

```diff
@@ -38,6 +39,10 @@ uInt32 ColourPalette::getRGB(int val) const {
     return m_palette[val];
 }
 
+void ColourPalette::getPaletteCopy(int palette_copy[256])
+{
+    memcpy(palette_copy, m_palette, 256*sizeof(int));
```

Comment:
Style suggestion: you can directly write `sizeof(palette_copy)` here to avoid duplicating the constant '256'.